apps/artemis-web/pages/api/rmq_action.ts

```diff
@@ -0,0 +1,150 @@
+import authorization from '../../middleware/authorization';
+import nc from 'next-connect';
+import {
+  NextApiRequestExtended,
+  NextApiResponseExtended,
+} from '../../definitions';
+import auth from '../../middleware/auth';
+import * as Amqp from 'amqp-ts';
+import { BrokerExchangeOptions, BrokerQueueOptions } from 'typescript-rabbitmq';
+import Broker from 'typescript-rabbitmq';
+
+const sendRMQAction = async (obj) => {
+  const { exchangeName, payload, routing_key } = obj;
+  const {
+    RABBITMQ_USER,
+    RABBITMQ_PASS,
+    RABBITMQ_HOST,
+    RABBITMQ_PORT,
+  } = process.env;
+
+  let config: any = {
+    connection: {
+      user: RABBITMQ_USER,
+      pass: RABBITMQ_PASS,
+      host: '172.18.0.16',
+      port: RABBITMQ_PORT,
+      timeout: 2000,
+      name: 'rabbitmq',
+    },
+    exchanges: [],
+    queues: [],
+    binding: [],
+    logging: {
+      adapters: {
+        stdOut: {
+          level: 3,
+          bailIfDebug: true,
+        },
+      },
+    },
+  };
+
+  let broker = new Broker(config);
+  await broker.connect();
+
+  await broker.addExchange(exchangeName, 'direct', {
+    durable: false,
+  } as BrokerExchangeOptions);
+  await broker.init();
+
+  console.log('sender: going to send message ..');
+  await broker.send(exchangeName, routing_key, payload.toString());
+  console.log('sender: message sent ..');
+};
+
+const handler = nc()
+  .use(auth)
+  .use(authorization(['admin']))
+  .post(async (req: NextApiRequestExtended, res: NextApiResponseExtended) => {
+    let obj = {};
+    const {
+      action,
+      hijack_key,
+      prefix,
+      hijack_type,
```

Comment:
I think this should be just "type", otherwise there is a mismatch in the expected payload on the other side.